src/teacher.py

```diff
@@ -21,9 +21,13 @@ def __init__(self, train_tasks: Sequence[Task]) -> None:
         env = create_env(CFG.env)
         self._pred_name_to_pred = {pred.name: pred for pred in env.predicates}
         self._allowed_query_type_names = get_allowed_query_type_names()
-        self._oracle_approach = create_approach("oracle", env.predicates,
-                                                env.options, env.types,
-                                                env.action_space, [])
+        self._oracle_approach = OracleApproach(
```

Comment:
should we remove the "oracle" `if` statement from create_approach entirely, so we don't have this issue again in the future?

Reply:
oh shoot but that would make `--oracle` crash so probably a bad idea lol